libs/dal/src/repositories/organization/organization.entity.ts

```diff
@@ -28,6 +28,8 @@ export class OrganizationEntity implements IOrganizationEntity {
 
   externalId?: string;
 
+  stripeCustomerId?: string;
+
```

Comment:
This is not ideal - we should probably think of a way to split enterprise and community entities+schema so its clear which attributes are available using EE and community repositories, rather than marking them optional.

Will create an action item to address this.

Reply:
Great call out! I agree 100%.